src/dotnet/commands/dotnet-pack/NuGet/ManifestVersionUtility.cs

```diff
@@ -15,9 +15,6 @@ internal static class ManifestVersionUtility
         public const int TargetFrameworkSupportForDependencyContentsAndToolsVersion = 4;
         public const int TargetFrameworkSupportForReferencesVersion = 5;
         public const int XdtTransformationVersion = 6;
-        // Note that this version should change from 7 to 8 when the PackageType
-        // schema is merged into here
-        public const int ServiceableVersion = 7;
 
         public static int GetManifestVersion(ManifestMetadata metadata)
         {
```

Comment:
I think we missed one comment from https://github.com/dotnet/cli/pull/3492/files

```
             // Important: always add newer version checks at the top
```